src/components/content/heroes/heroes.stories.mdx

```diff
@@ -1,63 +1,8 @@
-import { Canvas, Meta } from '@storybook/addon-docs';
-import { AuroraHero } from './aurora-hero'; import { AuroraType } from './aurora/aurora';
+import { Meta } from '@storybook/addon-docs';
 
 <Meta
   title="Components/Content/Heroes" />
 
-# Aurora Hero
```

Comment:
In hindsight we had good ambitions creating storybook content for our website but I doubt folks had using or maintained it properly 😬  Maybe we can remove storybook content entirely over time